frontend/viewer/src/lib/components/stomp/stomp-safe-lcm-rich-text-editor.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+  import type { ComponentProps } from 'svelte';
+  import {StompGuard} from './stomp-guard.svelte';
+  import {mergeProps} from 'bits-ui';
+  import LcmRichTextEditor from '../lcm-rich-text-editor/lcm-rich-text-editor.svelte';
+
+  type Props = ComponentProps<typeof LcmRichTextEditor>;
+
+  let { value = $bindable(), ...rest}: Props = $props();
+
+  const guardedValue = new StompGuard(
+    () => value,
+    (newValue) => value = newValue
+  );
+</script>
+
+<LcmRichTextEditor bind:value={guardedValue.value} {...mergeProps({
```

Comment:
`guardedValue.value` is awkward. What about either `guarded.value` or guardedValue.current`?

Reply:
I went with `guard.value`.